README.md

```diff
@@ -5,10 +5,8 @@ This bundle provides various tools to rapidly develop RESTful API's &
 applications with Symfony. Features include:
```

Comment:
you could list the exception.messages and the rfc compatible error rendering that works with both the symfony/serializer as well as jms